x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/ClientTransformIndexer.java

```diff
@@ -433,24 +445,26 @@ private void injectPointInTimeIfNeeded(SearchRequest searchRequest, ActionListen
                         e
                     );
                 }
-                listener.onResponse(searchRequest);
+                listener.onResponse(namedSearchRequest);
             })
         );
     }
 
-    private void doSearch(SearchRequest searchRequest, ActionListener<SearchResponse> listener) {
-        logger.trace("searchRequest: {}", searchRequest);
+    private void doSearch(Tuple<String, SearchRequest> namedSearchRequest, ActionListener<SearchResponse> listener) {
+        logger.trace("searchRequest: {}", namedSearchRequest.v2());
```

Comment:
This may serialize `namedSearchRequest.v2()` string even when trace is not turned on correct? That could be expensive.